pom.xml

```diff
@@ -440,6 +440,26 @@
       <version>${jedis.version}</version>
       <type>jar</type>
     </dependency>
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka-streams</artifactId>
+      <version>${kafka.version}</version>
+      <classifier>test</classifier>
```

Comment:
why is this dependency required?

Reply:
I tried to get the EmbeddedKafkaCluster to work for unit tests. Will remove this dependency.